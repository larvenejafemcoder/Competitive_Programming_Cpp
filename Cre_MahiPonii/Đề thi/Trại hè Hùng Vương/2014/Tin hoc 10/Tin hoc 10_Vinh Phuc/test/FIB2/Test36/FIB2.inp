100
33 2499417
39 60864203
21 1477
18 2696
3 2
38 10134147
37 21751875
45 1163502166
33 5240146
5 8
40 39766109
0 1
34 1544335
24 68114
31 1152455
21 6294
9 47
2 2
41 72923145
28 100553
35 7993271
26 32255
16 281
16 325
36 4056561
30 1334435
30 309645
42 182108563
21 3249
42 328357983
13 76
3 1
22 12687
30 1344948
8 30
18 2693
19 4679
44 492032016
36 18825188
29 339256
39 87255209
23 6316
16 664
39 12173394
37 6361382
16 471
36 3684366
4 3
18 4010
19 1844
13 346
27 193774
27 152908
12 90
35 13238982
38 39091275
18 1979
5 8
40 59432838
44 999209973
43 131068966
10 51
37 35420329
3 3
3 2
38 36209695
20 263
39 77664739
22 5000
29 358901
45 261246060
2 1
1 1
45 1699191622
0 1
9 20
38 61406973
9 48
45 46770514
23 4022
16 264
23 21324
5 8
10 46
10 87
32 3029929
37 17659954
28 361732
27 29317
37 29091912
36 1970504
29 113118
35 4684608
14 79
29 59081
6 10
9 47
24 1785
8 7
26 163628
