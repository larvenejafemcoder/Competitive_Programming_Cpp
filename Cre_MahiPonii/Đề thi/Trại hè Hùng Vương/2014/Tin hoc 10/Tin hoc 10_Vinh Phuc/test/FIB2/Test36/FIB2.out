954692
23248057
564
1030
1
3870900
8308477
444418282
2001558
3
15189302
1
589884
26017
440199
2404
18
1
27854163
38408
3053158
12320
107
124
1549469
509709
118274
69559282
1241
125421589
29
0
4846
513725
12
1029
1787
187939507
7190582
129584
33328524
2413
254
4649823
2429832
180
1407303
1
1532
704
132
74015
58406
34
5056841
14931539
756
3
22701324
381664248
50063890
20
13529362
1
1
13830873
101
29665291
1910
137088
99787116
1
0
649033446
1
8
23455377
18
17864747
1536
101
8145
3
18
33
1157330
6745502
138169
11198
11112122
752666
43207
1789361
30
22567
4
18
682
3
62500
